packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -164,17 +173,85 @@ export const TransactionReviewModalContent = ({
         reportTransactionCreatedEvent('downloaded');
     };
 
+    const BottomContent = () => {
+        if (isRbfConfirmedError) {
+            return (
+                <NewModal.Button variant="tertiary" onClick={onCancel}>
+                    <Translation id="TR_CLOSE" />
+                </NewModal.Button>
+            );
+        }
+
+        if (!areDetailsVisible && isBroadcastEnabled) {
```

Comment:
When I look at the original condition it says
`!areDetailsVisible && (isBroadcastEnabled ? <JSX1 /> : <JSX2 />)`
You did not keep the same logic, is this intentional?
I believe this upholds original behavior:
```suggestion
        if (areDetailsVisible) return null; 
        if (isBroadcastEnabled) {
```

Reply:
yes, you're right, nice catch, thank you, fixed